src/ValueObject/PhpVersionFeature.php

```diff
@@ -689,4 +689,11 @@ final class PhpVersionFeature
      * @var int
      */
     public const DEPRECATE_HOST_PORT_SEPARATE_ARGS = PhpVersion::PHP_83;
+
+    /**
+     * @see https://www.php.net/manual/en/migration83.deprecated.php#migration83.deprecated.core.get-class
+     * @see https://php.watch/versions/8.3/get_class-get_parent_class-parameterless-deprecated
+     * @var int
+     */
+    public const DEPRECATE_GET_CLASS_WITHOUT_ARGS = PhpVersion::PHP_83;
```

Comment:
create 2 constants for 2 rules for differentiate functionality.

Reply:
eg, add new constant: `DEPRECATE_GET_PARENT_CLASS_WITHOUT_ARGS`

Reply:
ah, I thought the mapping here was per php deprecation, not per rector. Would it make more sense to combine the two rectors into a single one? Much like how the utf-8 en/de-code deprecation was handled?

Reply:
I see, combine to single rule is fine for me, since only 2 functions